master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md

```diff
@@ -18,6 +18,7 @@ You must have a cluster which meets the following requirements:
 
 - You have a Kubernetes cluster configured to use CNI network plugins (i.e by passing --network-plugin=cni to the kubelet)
 - Your Kubernetes controller manager is configured to allocate pod CIDRs (i.e by passing --allocate-node-cidrs=true to the controller manager)
```

Comment:
I also noticed the use of `i.e`, Is that a common variation of `i.e.`?  A quick google search didn't find any usages without the trailing period so I think we should use the `i.e.` version.